redash/serializers/query_result.py

```diff
@@ -57,6 +57,14 @@ def _get_column_lists(columns):
     return fieldnames, special_columns
 
 
+def serialize_query_result(query_result, is_api_user):
+    if is_api_user:

```

Comment:
It's safe to use `current_user` in the serialization code instead of passing `is_api_user`. It's not a big deal for this PR, but #3782 will extend this method to apply other checks anyway.